csrankings-f.csv

```diff
@@ -135,8 +135,6 @@ Fatih Alagöz,Boğaziçi University,https://www.cmpe.boun.edu.tr/~alagoz,5Nku4S8
 Fatima A. Merchant,University of Houston,https://www.uh.edu/technology/departments/et/people/?l=merchant&f=fatima,fUCDUOoAAAAJ
 Fatima Abu Salem,American University of Beirut,http://www.cs.aub.edu.lb/fa21,NOSCHOLARPAGE
 Fatima K. Abu Salem,American University of Beirut,http://www.cs.aub.edu.lb/fa21,NOSCHOLARPAGE
-Fatos T. Yarman-Vural,Middle East Technical University,http://www.ceng.metu.edu.tr/~vural,zPFQ2BwAAAAJ
-Fatos Tünay Yarman Vural,Middle East Technical University,http://www.ceng.metu.edu.tr/~vural,zPFQ2BwAAAAJ

```

Comment:
Please point to something indicating change of affiliation for this person.

Reply:
She is a former faculty member as indicated here -> https://ceng.metu.edu.tr/former-faculty-members